vulcain.go

```diff
@@ -152,9 +153,10 @@ func (v *Vulcain) IsValidRequest(req *http.Request) bool {
 
 // IsValidResponse checks if Apply will be able to deal with this response.
 func (v *Vulcain) IsValidResponse(req *http.Request, responseStatus int, responseHeaders http.Header) bool {
-	// Not a success, or not JSON: don't modify the response
+	// Not a success,marked as no-transform or not JSON: don't modify the response
 	if responseStatus < 200 ||
 		responseStatus > 300 ||
+		notransformRe.MatchString(responseHeaders.Get("Cache-Control")) ||
```

Comment:
I suggest to move this test after the one on the content type because it's most lively that the content isn't JSON than this attribute is set.

Reply:
Okay, will update.